src/components/atoms/MunicipalitySelector.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { SelectField } from 'evergreen-ui';
+
+type Props = {
```

Comment:
I think we should add a nullable prop for selected municipality in case the user clicks the edit button from the Dashboard, so the correct municipality is pre-selected.

Reply:
This is part of #126 and #131